k2/python/k2/fsa_algo.py

```diff
@@ -1011,3 +1011,78 @@ def ctc_graph(symbols: Union[List[List[int]], k2.RaggedInt],
     aux_labels = k2.index(symbol_values, arc_map)
     fsa = Fsa(ragged_arc, aux_labels=aux_labels)
     return fsa
+
+def ctc_topo(max_token: int, standard: bool) -> k2.Fsa:
```

Comment:
I think it would be clearer to have 'modified: bool = False' as the arg.

Reply:
Thanks. Fixed.